README.md

```diff
@@ -4,41 +4,44 @@
-    * Tools to compare, merge, and publish databases using this tool
+  * Generate Steam Workshop Database (SteamDB) on the fly. This is compatible with & synonymous to Paladin's RimPy Community Mod Manager Database db.json schema
+  * Tools to compare, merge, and publish databases using this tool
 * Rule Editor for configured Community Rules database, as well as User Rules
-    * Fully compatible with Paladin's RimPy Community Mod Manager Database communityRules.json schema
-    * Tools to compare, merge, and publish databases using this tool
+  * Fully compatible with Paladin's RimPy Community Mod Manager Database communityRules.json schema
+  * Tools to compare, merge, and publish databases using this tool
 * Support for creating additional sorting modes (in code)
     * "Alphabetical" sorting algorithm
     * "Topological" sorting algorithm
 
-To run RimSort, visit the [Releases](https://github.com/oceancabbage/RimSort/releases) page and download the latest zipped release for your operating system. For Windows and Linux, unzip the download and run the `RimSort` executable inside the unzipped folder. For MacOS, unzip the download and run the `.app` directly. For more information on how to run and use RimSort, visit the [User Guide](https://github.com/oceancabbage/RimSort/wiki/User-Guide).
+To run RimSort, visit the [Releases](https://github.com/RimSort/RimSort/releases) page and download the latest zipped release for your operating system. For Windows and Linux, unzip the download and run the `RimSort` executable inside the unzipped folder. For MacOS, unzip the download and run the `.app` directly. For more information on how to run and use RimSort, visit the [User Guide](https://github.com/RimSort/RimSort/wiki/User-Guide)
 
-RimSort is currently under active development. Contributors are welcome! If you are interested in helping develop RimSort, read the [Development Guide](https://github.com/oceancabbage/RimSort/wiki/Development-Guide) on to get started-- this Wiki contains lots of information on how RimSort works under the hood.
+RimSort is currently under active development. Contributors are welcome! You can read about how you can help develop RimSort in the [Development Guide](https://github.com/RimSort/RimSort/wiki/Development-Guide)
+This Wiki contains lots of information on how RimSort works under the hood.
 
-There are lots of planned features for RimSort. Most of them are tracked in the [Issues](https://github.com/oceancabbage/RimSort/issues) section of this repo. If you have a feature suggestion, feel free to create an Issue here yourself!
+There are lots of planned features for RimSort. Most of them are tracked in the [Issues](https://github.com/RimSort/RimSort/issues) section of this repo.
+If you have a feature suggestion, feel free to create an Issue yourself! Submit a new feature request by using the issue template, but please check the existing features first!
```

Comment:
Please change:

* "Submit a new feature request by using the issue template, but please check the existing features first!"

to

* "Submit a new feature request by using the corresponding Issue template, but please check existing issues first to avoid duplicate requests!"